apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingStatusService.kt

```diff
@@ -289,6 +295,22 @@ class BehandlingStatusServiceImpl(
         }
     }
 
+    private fun haandterFeilutbetaling(behandling: Behandling) {
+        val brevutfall = behandlingInfoDao.hentBrevutfall(behandling.id)
+        if (brevutfall?.feilutbetaling?.valg in listOf(FeilutbetalingValg.JA_VARSEL, FeilutbetalingValg.JA_INGEN_TK)) {
```

Comment:
Gir det meining at dette skal skje for ja_ingen_tk?

Reply:
@madsop-nav : Skjønner at det virker litt rart, men det er feilutbetaling her bare at den er under 4 rettsgebyr. Dette må svares ut mot økonomi - men vi skal ikke sende dette som et vedtak til bruker. Med andre ord får saksbehandler en oppgave her som må gjøres.

Reply:
Ah 💡